templates/manifest.j2

```diff
@@ -94,16 +94,8 @@ ram.runtime = "50M"
     [resources.sources]
 
         [resources.sources.main]
-        # This will pre-fetch the asset which can then be deployed during the install/upgrade scripts with :
-        #    ynh_setup_source --dest_dir="$install_dir"
-        # You can also define other assets than "main" and add --source_id="foobar" in the previous command
         url = "{{data.source_url}}"
-        sha256 = "{{data.sha256sum}}"
-
-        # These infos are used by https://github.com/YunoHost/apps/blob/master/tools/autoupdate_app_sources/autoupdate_app_sources.py
-        # to auto-update the previous asset urls and sha256sum + manifest version
-        # assuming the upstream's code repo is on github and relies on tags or releases
-        # See the 'sources' resource documentation for more details
+        sha256 = "sha256sum"
```

Comment:
Hmmm if the point is to compute it automagically, we'll need a different syntax here such as

```suggestion
        sha256 = "{{data.sha256sum}}"
```

(but have it autocomputed in the code)